src/PointEvalHandler.jl

```diff
@@ -58,7 +59,13 @@ end
 function PointEvalHandler(grid::AbstractGrid{dim}, points::AbstractVector{Vec{dim, T}}; search_nneighbors = 3, warn::Bool = true, strategy = NewtonLineSearchPointFinder()) where {dim, T}
     node_cell_dicts = _get_node_cell_map(grid)
     cells, local_coords = _get_cellcoords(points, grid, node_cell_dicts, search_nneighbors, warn, strategy)
-    return PointEvalHandler(grid, cells, local_coords)
+    return PointEvalHandler(grid, cells, local_coords, false)
+end
+
+function PointEvalHandler(grid::AbstractGrid{dim}, points::AbstractVector{Vec{dim, T}}, subdomains::Vector{OrderedSet{Int}}; search_nneighbors = 3, warn::Bool = true, strategy = NewtonLineSearchPointFinder()) where {dim, T}
```

Comment:
Would be nice if we could have a keword argument `cellset` here instead of `subdomains`, see my comment below.

Reply:
I agree that a kwarg makes the user-experience better and we don't lock in on arg ordering. 

Even if an arg defaulting to `1:getncells(grid)` is alright from the convention that args say what and kwargs say how.